client/packages/programs/src/JsonForms/components/Search/useSearchQueries.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { getPatientQueries } from 'packages/system/src/Patient/api/api';
+import { getSdk } from 'packages/system/src/Patient/api/operations.generated';
+import { useGql, useAuthContext, Typography } from '@openmsupply-client/common';
+import { RegexUtils, JSXFormatters } from '@openmsupply-client/common';
+
+export const QueryValues = ['patientByCode'] as const;
```

Comment:
First think I tried was searching by name :) think we need to support this (search by code OR firstName OR lastName) but this might need another change to the BE api...

Reply:
Yeah, the issue (#1070) specifically asks for searching by patient code. Can other searches, but this would be a new issue I think (when it's required).

Reply:
Ok cool, lets see what @richardmoizeau thinks. After having another look at the current implementation I have the feeling we need to support manual mother entries as well...